src/main/controllers/TribunalOrderOrRequestDetailsController.ts

```diff
@@ -31,10 +31,14 @@ export default class TribunalOrderOrRequestDetailsController {
     }
 
     const redirectUrl =
-      PageUrls.TRIBUNAL_RESPOND_TO_ORDER.replace(':orderId', req.params.orderId) + getLanguageParam(req.url);
+      selectedRequestOrOrder.value.sendNotificationCaseManagement === 'Request'
+        ? PageUrls.PREPARE_DOCUMENTS.replace(':orderId', req.params.orderId) + getLanguageParam(req.url)
+        : PageUrls.TRIBUNAL_RESPOND_TO_ORDER.replace(':orderId', req.params.orderId) + getLanguageParam(req.url);
```

Comment:
minor: The `.replace(...)` bit is the same for both lines, can this be refactored to avoid repetition?

Reply:
Refactored